namer/consul/src/main/scala/io/buoyant/namer/consul/LookupCache.scala

```diff
@@ -41,13 +42,21 @@ private[consul] class LookupCache(
     }
   }
 
-  // Note that this activity is never recomputed.  We simply do a
+  private[this] def resolveDc(Datacenter: String): Activity[String] = localDc match {
```

Comment:
I found this pretty confusing because `localDc` isn't actually the local dc, it's the moniker.